compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1953,3 +1953,6 @@ error.module.generated.for.client.decl.must.have.a.client.object.type=\
 
 error.module.generated.for.client.decl.cannot.have.mutable.state=\
   a module generated for a client declaration cannot have mutable state
+
+error.unsupported.alternative.wait.action.in.multiple.wait.expr=\
+  alternative wait action not yet support inside multiple wait expressions
```

Comment:
```suggestion
  cannot use an alternate wait action within a multiple wait action
```

Reply:
alternative -> alternate

Reply:
We say "not yet supported" only for stuff that is in the spec, but not in the implementation yet. Since this isn't in the spec yet, let's just give the error according to the current spec.

Reply:
Updated. Please check. 48185375e60cb5de6a89b97882502483cc1aea9c